code/SiteBanner.php

```diff
@@ -54,7 +55,8 @@ public function fieldLabels($includerelations = true)
                 'Content' => _t('SiteBanner.ContentFieldLabel', 'Banner content'),
                 'Type' => _t('SiteBanner.TypeFieldLabel', 'Banner type'),
                 'StartDate' => _t('SiteBanner.StartDateFieldLabel', 'Start date / time'),
-                'EndDate' => _t('SiteBanner.EndDateFieldLabel', 'End date / time')
+                'EndDate' => _t('SiteBanner.EndDateFieldLabel', 'End date / time'),
+                'Dismiss' => _t('SiteBanner.DismissLabel', 'Allow users to dismiss this banner')
```

Comment:
Add a description which states that banners are only temporarily dismissed in the current browser session